cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,44 @@
-describe('', () => {
+describe('This article test', () => {
+  let fakeuser;
   before(() => {
-
+    cy.visit('https://conduit.mate.academy');
```

Comment:
```suggestion
```
you do not need to visit page only once. Better to move it inside the test. Because first you make a request for login, this request sets the cookies and then you visit the page as logged in user